src/apps/AppCenter/InstalledApps/RepoVersions.js

```diff
@@ -11,8 +11,8 @@ import {
   useLayout,
   useTheme,
 } from '@aragon/ui'
-import { format } from 'date-fns'
 import { RepoType } from '../../../prop-types'
+import { dateFormat } from '../../../date-utils'
```

Comment:
This is a small nitpick, but I think we've generally preferred `formatX()` for names, so this would become `formatDate()`